Sources/FuzzilliCli/main.swift

```diff
@@ -349,6 +349,9 @@ func makeFuzzer(for profile: Profile, with configuration: Configuration) -> Fuzz
     // The evaluator to score produced samples.
     let evaluator = ProgramCoverageEvaluator(runner: runner, maxResetCount: UInt64(maxResetCount))
 
+    // Temporary to allow export
```

Comment:
Left over?

Reply:
Ahh my bad! I did a couple runs where I exported all edges from both a basicCorpus and a markovCorpus over 8 hours. I'll add the charts once I crunch the data